File: dpo_train/translation_output/java-py/HumanEval_79_decimal_to_binary.py
def decimal_to_binary(decimal: int) -> str:
    return decimalToBinary(decimal)

def check(candidate):
    assert candidate(0) == 'db0db'
    assert candidate(32) == 'db100000db'
    assert candidate(103) == 'db1100111db'
    assert candidate(15) == 'db1111db'

def test_check():
    check(decimal_to_binary)

test_check()

File: test/before_train/testset/py-cpp/HumanEval_79_decimal_to_binary.cpp
#include<assert.h>
#include<bits/stdc++.h>
#include<vector>
#include<sstream>
#include<regex>
#include<map>
#include<set>
#include<unordered_map>
#include<unordered_set>
std::string decimal_to_binary(long decimal) {
    std::stringstream ss;
    ss << decimal;
    return ss.str();
}
int main() {
    auto candidate = decimal_to_binary;
    assert(candidate((0)) == ("db0db"));
    assert(candidate((32)) == ("db100000db"));
    assert(candidate((103)) == ("db1100111db"));
    assert(candidate((15)) == ("db1111db"));
}
